tests/test_utils/utils.py

```diff
@@ -52,3 +60,50 @@ def mock_definitions():
         exodus_config = json.load(f)
 
     return exodus_config
+
+
+def encode_signature(data: bytes):
+    return (
+        base64.b64encode(data)
+        .replace(b"+", b"-")
+        .replace(b"=", b"_")
+        .replace(b"/", b"~")
+    )
+
+
+def build_policy(url: str, expiration: datetime):
+    datelessthan = int(datetime2timestamp(expiration))
+    condition = {"DateLessThan": {"AWS:EpochTime": datelessthan}}
+    # omit the Resource parameter, users can access all of the files
+    payload = [("Condition", condition)]
+    policy = {"Statement": [OrderedDict(payload)]}
+    return json.dumps(policy, separators=(",", ":")).encode("utf-8")
+
+
+def rsa_signer(private_key: str, policy: bytes):
+    bytes_key = bytes(private_key, "utf-8")
+    loaded_key = serialization.load_pem_private_key(
+        bytes_key, password=None, backend=default_backend()
+    )
+    return loaded_key.sign(policy, padding.PKCS1v15(), hashes.SHA1())  # type: ignore
+
+
+def generate_cookies():
+    # Env var for using signed cookies
+    CDN_PRIVATE_KEY = os.environ.get("EXODUS_CDN_PRIVATE_KEY")
```

Comment:
Minor: local vars shouldn't be named in all caps.  You'd name these CDN_PRIVATE_KEY etc if they were globals, which they aren't.